### PR TITLE
Fixing typos

### PR DESCRIPTION
**Pull request by identify1980**
Might be nitpicky at some points, but hopefully no-one minds ;-)

### CODE REVIEW

File: Source/EventFlow/Extensions/SpecificationExtensions.cs [hunk 1]
```diff
@@ -34,7 +34,7 @@ namespace EventFlow.Extensions
 {
     public static class SpecificationExtensions
     {
-        public static void ThrowDomainErrorIfNotStatisfied<T>(
+        public static void ThrowDomainErrorIfNotSatisfied<T>(
```
Comment (rasmus) [inline Source/EventFlow/Extensions/SpecificationExtensions.cs]: Please create an overload with the corrected name and put a `[Obsolete]` attribute on this as people are already using it

**Comment by identify1980** [comment]
OK, added the [Obsolete] attribute... hopefully in a correct way (it's my first contribution to an open source project).
I accepted the CLA. Unfortunately I cannot do this for my previous commits - I simply forgot to adjust my local gitsettings to match my GitHub account.

File: Source/EventFlow/Extensions/SpecificationExtensions.cs [hunk 2]
```diff
@@ -44,7 +45,21 @@ public static class SpecificationExtensions
             if (whyIsNotStatisfiedBy.Any())
             {
                 throw DomainError.With(
-                    $"'{specification.GetType().PrettyPrint()}' is not satisfied becase of {string.Join(" and ", whyIsNotStatisfiedBy)}");
+                    $"'{specification.GetType().PrettyPrint()}' is not satisfied because of {string.Join(" and ", whyIsNotStatisfiedBy)}");
+            }
+        }
+
+        public static void ThrowDomainErrorIfNotSatisfied<T>(
```
Comment (rasmus) [inline Source/EventFlow/Extensions/SpecificationExtensions.cs]: Please merely invoke the new from the obsoleted version, no need for the code duplication

Comment (identify1980) [reply Source/EventFlow/Extensions/SpecificationExtensions.cs]: Done

**Comment by rasmus** [comment]
@identify1980 Perfect, we got the CLA history here, got to be good enough. Thanks again for the PR.

**Pull request merged**